app/models/foreman_ansible/ansible_provider.rb

```diff
@@ -32,7 +32,8 @@ def proxy_command_options(template_invocation, host)
               template_invocation.template
             ),
             :name => host.name,
-            :check_mode => host.host_param('ansible_roles_check_mode')
+            :check_mode => host.host_param('ansible_roles_check_mode'),
+            :job_invocation_timeout => template_invocation.job_invocation[:execution_timeout_interval]
```

Comment:
why do we need this, it seems to be sent as `execution_timeout_interval` already, or is it not?